sentinel.py

```diff
@@ -4,31 +4,50 @@
 """
 Create sentinel and singleton objects.
 
-Copyright 2014 © Eddie Antonio Santos. MIT licensed.
+ Copyright 2014 © Eddie Antonio Santos. MIT licensed.
+
+ This software is released under the MIT License.
+ https://opensource.org/licenses/MIT
 """
 
 import inspect
+from typing import Dict, Tuple, Optional, Any
 
 __all__ = ['create']
-__version__ = '0.1.2'
+__version__ = '0.1.3'
```

Comment:
Strictly speaking as far as SemVer goes, this is adding a new feature, and added a breaking change (renaming a keyword argument `extra_methods` -> `cls_dict` is a breaking change) this this should be a minor version bump.

```suggestion
__version__ = '0.2.0'
```

See: https://semver.org/#spec-item-4